src/RenderWebGL.js

```diff
@@ -757,13 +761,15 @@ class RenderWebGL extends EventEmitter {
     /**
      * Check if a particular Drawable is touching any in a set of Drawables.
      * @param {int} drawableID The ID of the Drawable to check.
-     * @param {?Array<int>} candidateIDs The Drawable IDs to check, otherwise all drawables in the renderer
+     * @param {?Array<int>} candidateIDs The Drawable IDs to check, otherwise all visible drawables in the renderer
      * @returns {boolean} True if the Drawable is touching one of candidateIDs.
      */
     isTouchingDrawables (drawableID, candidateIDs = this._drawList) {
-
-        const candidates = this._candidatesTouching(drawableID, candidateIDs);
-        if (candidates.length === 0) {
+        const candidates = this._candidatesTouching(drawableID,
+            // even if passed an invisible drawable, we will NEVER touch it!
+            candidateIDs.filter(id => this._allDrawables[id]._visible));
```

Comment:
Should this actually just be _visibleDrawList? I'm actually not sure if I understand the old comment for this function in the context of the old code. Was candidateIDs supposed to be an optional param but then it got turned into a required param? If it is a required param, then neither of these sentences makes sense to me:

> -The Drawable IDs to check, otherwise all drawables in the renderer

> +The Drawable IDs to check, otherwise all visible drawables in the renderer

Reply:
So the problem here is you can pass it an invisible drawable, `isTouchingDrawables` should never touch an invisible drawable - Even if you say "am i touching this hidden drawable" the answer is "no" -- The unit tests actually test for this case, so I'm confident it's doing the right thing